org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/SlicingExamples.java

```diff
@@ -43,6 +45,112 @@ public void before() {
 	}
 
 	/**
+	 * Slice using basic 1D int array. The goal is to take the data from index 1
+	 * This should display : [2,3]
+	 */
+	@Test
+	public void sliceFrom1DFromIndex1() {
+		Dataset onedData = DatasetFactory.createFromObject(new int[] { 1, 2, 3 });
+		Dataset sliceData = onedData.getSlice(new Slice(1, null, null));
+		System.out.println(sliceData.toString(true));
+	}
+
+	/**
+	 * Slice using basic 1D int array. The goal is to create a Slice with only
+	 * the stop index
+	 * This should display : [1,2]
+	 */
+	@Test
+	public void sliceFrom1DWithStopIndex() {
+		Dataset onedData = DatasetFactory.createFromObject(new int[] { 1, 2, 3 });
+		Dataset sliceData = onedData.getSlice(new Slice(2));
+		System.out.println(sliceData.toString(true));
+	}
+
+	/**
+	 * Slice using basic 1D int array. The goal is to get the middle of the
+	 * array
+	 * This should display : [6, 5, 4]
+	 */
+	@Test
+	public void sliceFrom1D() {
+		Dataset onedData = DatasetFactory.createFromObject(new int[] { 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0 });
+		// we want to get the 3 values middle of the array, that's to say 6,5,4
```

Comment:
Simplify the comment to say

'We want to get the 4th to the 6th index. Start argument is inclusive so this will be 4. Stop argument is exclusive so this will be 7.'